### PR TITLE
[244] address sonar issues in wildfly/cdk bundles

### PR DESCRIPTION
**Pull request by adietish**
Signed-off-by: Andre Dietisheim <adietish@redhat.com>

### CODE REVIEW

File: framework/bundles/org.jboss.tools.rsp.foundation.core/src/main/java/org/jboss/tools/rsp/foundation/core/transport/URLTransportCache.java [hunk 1]
```diff
@@ -52,7 +52,7 @@
 	private static final Logger LOG = LoggerFactory.getLogger(URLTransportCache.class);
 
 	private static final int DEFAULT_CONNECT_TIMEOUT = 1 * 60 * 1000;
-	private static final int DEFAULT_READ_TIMEOUT = 2 * 60 * 1000;
+	private static final int DEFAULT_READ_TIMEOUT = (int)(0.1 * 60 * 1000);
```
Comment (robstryker) [inline framework/bundles/org.jboss.tools.rsp.foundation.core/src/main/java/org/jboss/tools/rsp/foundation/core/transport/URLTransportCache.java]: Is this intentional, to change the read timeout from 60 seconds to 6 seconds?

File: runtimes/bundles/org.jboss.tools.rsp.server.minishift/src/main/java/org/jboss/tools/rsp/server/minishift/discovery/MinishiftBeanType.java [hunk 2]
```diff
@@ -75,10 +75,9 @@ private MinishiftVersions findOrLoad(File root) {
 	@Override
 	public String getUnderlyingTypeId(File root) {
 		MinishiftVersions props = findOrLoad(root);
-		if( props != null ) {
-			if( props.getCDKVersion() != null ) {
-				return "CDK";
-			}
+		if( props != null 
+				&& props.getCDKVersion() != null ) {
```
Comment (robstryker) [inline runtimes/bundles/org.jboss.tools.rsp.server.minishift/src/main/java/org/jboss/tools/rsp/server/minishift/discovery/MinishiftBeanType.java]: couldn't this be on one line?  if( props != null && props.getCDKVersion() != null ) {

File: runtimes/bundles/org.jboss.tools.rsp.server.wildfly/src/main/java/org/jboss/tools/rsp/server/wildfly/runtimes/download/DownloadManagerWorkflowUtility.java [hunk 3]
```diff
@@ -55,7 +60,8 @@ public static int getWorkflowStatus(DownloadRuntime dr, String userS, String pas
 		if (response == 401) {
 			// 401 means bad credentials, change nothing
 			return CREDENTIALS_FAILED;
-		} else if (response == 403 || response == 200) {
+		} else if (response == 403
+				|| response == 200) {
```
Comment (robstryker) [inline runtimes/bundles/org.jboss.tools.rsp.server.wildfly/src/main/java/org/jboss/tools/rsp/server/wildfly/runtimes/download/DownloadManagerWorkflowUtility.java]: Why does it move this to a new line? It's not even very long :(   I hate automatic code formatters :(

**Comment by robstryker** [comment]
If you just verify the DEFAULT_READ_TIMEOUT   situation, I can +1.  That timeout change seems iffy to me though.

File: framework/bundles/org.jboss.tools.rsp.foundation.core/src/main/java/org/jboss/tools/rsp/foundation/core/transport/URLTransportCache.java (hunk 1 above)
Comment (adietish) [reply framework/bundles/org.jboss.tools.rsp.foundation.core/src/main/java/org/jboss/tools/rsp/foundation/core/transport/URLTransportCache.java]: no wasnt, correcting. thing that this was when I started writing tests and forget to discard it.

File: runtimes/bundles/org.jboss.tools.rsp.server.minishift/src/main/java/org/jboss/tools/rsp/server/minishift/discovery/MinishiftBeanType.java (hunk 2 above)
Comment (adietish) [reply runtimes/bundles/org.jboss.tools.rsp.server.minishift/src/main/java/org/jboss/tools/rsp/server/minishift/discovery/MinishiftBeanType.java]: I prefer to separate both condition to the max and thus having every condition on a separate line

File: runtimes/bundles/org.jboss.tools.rsp.server.wildfly/src/main/java/org/jboss/tools/rsp/server/wildfly/runtimes/download/DownloadManagerWorkflowUtility.java (hunk 3 above)
Comment (adietish) [reply runtimes/bundles/org.jboss.tools.rsp.server.wildfly/src/main/java/org/jboss/tools/rsp/server/wildfly/runtimes/download/DownloadManagerWorkflowUtility.java]: this, too. On purpose. Like you prefer not to immediately return a value but to store in a temporary variable and only return afterwards: I prefer to have every condition on it's own line and get better readable imho.

**Comment by adietish** [comment]
@robstryker corrected the timeout, please re-review.

**Pull request merged**